src/services/cartsService.js

```diff
@@ -1,27 +1,35 @@
-const {
-  selectProdcutOptionId,
-  checkIfSameProduct,
-  insertProduct,
-  getCarts,
-  addQuantity,
-} = require('../models/cartsDao');
+const cartsDao = require('../models/cartsDao');
 
-const addItemToCartsService = async (userId, productId, sizeId) => {
-  const select = await selectProdcutOptionId(productId, sizeId);
+const addItemToCarts = async (userId, productId, sizeId) => {
+  const select = await cartsDao.selectProdcutOptionId(productId, sizeId);
   const productOptionId = select[0].id;
-  const check = await checkIfSameProduct(userId, productOptionId);
+  const check = await cartsDao.checkIfSameProduct(userId, productOptionId);
 
   if (check[0].check_product == 0) {
-    await insertProduct(userId, productOptionId);
+    await cartsDao.insertProduct(userId, productOptionId);
     return true;
   }
 
-  await addQuantity(userId, productOptionId);
+  await cartsDao.addQuantity(userId, productOptionId);
   return false;
 };
 
-const getCartsService = async (userId) => {
-  return await getCarts(userId);
+const getCarts = async (userId) => {
+  return await cartsDao.getCarts(userId);
+};
+
+const deleteProduct = async (userId, productOptionId) => {
+  const ifDeleted = await cartsDao.deleteCart(userId, productOptionId);
+  if (ifDeleted == 0) {
+    const err = new Error('product is not in carts');
+    err.status = 400;
+    throw err;
+  }
+  return 'deleted';
```

Comment:
```suggestion
  const result = await cartsDao.deleteCart(userId, productOptionId);
  if (result == 0) {
    const err = new Error('product is not in carts');
    err.status = 400;
    throw err;
  }
  return result;
```

업데이트 혹은 삭제에 대해서는 보통 그 요청에 맞는 결과를 반환해주는 것이 좋습니다.